website/docs/best-practices/how-we-mesh/mesh-4-faqs.md

```diff
@@ -0,0 +1,314 @@
+
+</detailsToggle>
+
+## How dbt Mesh works
+
+<detailsToggle alt_header="Can dbt Mesh handle cyclic dependencies between projects?">
+
+Like resource dependencies, project dependencies are acyclic, meaning they only move in one direction. This prevents `ref` cycles (or loops). For example, if project B depends on project A, a new model in project A could not import and use a public model from project B. Refer to [Project dependencies](/docs/collaborate/govern/project-dependencies#how-to-use-ref) for more information.
+
+</detailsToggle>
+
+<detailsToggle alt_header="Is it possible for multiple projects to directly reference a shared source?">
+
+While it’s not currently possible to share sources across projects, it would be possible to have a shared foundational project, with staging models on top of those sources, exposed as “public” models to other teams/projects.
+
+</detailsToggle>
+
+<detailsToggle alt_header="What if a model I've already built on from another project later becomes protected?">
+
+This would be a breaking change for downstream consumers of that model. If the maintainers of the upstream project wish to remove the model (or “downgrade” its access modifier, effectively the same thing), they should mark that model for deprecation (using [deprecation_date](/reference/resource-properties/deprecation_date)), which will deliver a warning to all downstream consumers of that model.
+
+In the future, we plan for dbt Cloud to also be able to proactively flag this scenario in [continuous integration](/docs/deploy/continuous-integration) for the maintainers of the upstream public model.
+
+</detailsToggle>
+
+<detailsToggle alt_header="If I run `dbt build --select +model`, will this trigger a run of upstream models in other projects?">
+
+No, unless downstream projects are installed as [packages](/docs/build/packages) (source code). In that case, the models in project installed as a project become “your” models, and you can select or run them. There are cases in which this can be desirable; see docs on [project dependencies](/docs/collaborate/govern/project-dependencies).
+
+</detailsToggle>
+
+<detailsToggle alt_header="If each project/domain has its own data warehouse, is it still possible to build models across them?">
+
+Yes; as long as they’re in the same data platform (such as BigQuery, Databricks, Redshift, Snowflake, or Starburst) and you have configured permissions and sharing in that data platform provider to allow for this.
+
+</detailsToggle>
+
+<detailsToggle alt_header="Can I run tests that involve tables from multiple different projects?">
+
+Yes! because the cross-project collaboration is done using the `{{ ref() }}` macro, you can use those models from other teams in [singular tests](/docs/build/data-tests#singular-data-tests). 
```

Comment:
```suggestion
Yes, because the cross-project collaboration is done using the `{{ ref() }}` macro, you can use those models from other teams in [singular tests](/docs/build/data-tests#singular-data-tests). 
```